amperity_api/source/endpoint_get_profiles_list.rst

```diff
@@ -23,6 +23,12 @@ Use the **GET /indexes/{id}/profiles** endpoint to return a paginated list of cu
 
 .. endpoint-get-profiles-list-end
 
+.. endpoint-get-profiles-list-maximum-filters-start
+
+.. note:: The maximum number of fields in an index is "30".
```

Comment:
It seems strange to put this in quotes unless that's a convention we've established.